yi-core/yi-core.cabal

```diff
@@ -135,3 +136,54 @@ library
       System.CanonicalizePath
       Yi.Buffer.Implementation
       Paths_yi_core
+
+test-suite tasty
```

Comment:
That was just an example, I haven't checked out this branch.

I didn't realize that the two package.yamls were crafted differently (this one was fine).